client/galaxy/scripts/mvc/tool/tool-form-base.js

```diff
@@ -66,14 +66,14 @@ define( [ 'utils/utils', 'utils/deferred', 'mvc/ui/ui-misc', 'mvc/form/form-view
             // build request url
             var build_url = '';
             var build_data = {};
-            if ( new_options.job_id ) {
-                build_url = Galaxy.root + 'api/jobs/' + new_options.job_id + '/build_for_rerun';
+            if ( options.job_id ) {
+                build_url = Galaxy.root + 'api/jobs/' + options.job_id + '/build_for_rerun';
```

Comment:
These 2 changes above seem to revert the recent commit https://github.com/galaxyproject/galaxy/commit/a498390bef74301816945551a9832f38747fd4fc from @guerler which I think was correct, still a bad merge maybe?